### ISSUE TITLE
<BANKID /><ACCTID /> remain blank 

### ISSUE THREAD

**Issue by jomajojo** [3v1n0/ofxstatement-n26#3]
I am using n26 plugin.
my config.ini is :
[n26]
plugin = n26
currency = EUR
bank_id = N26 Bank
account = DE12345678910

after generating the ofx file the statements BANKID and ACCTID are like in the title , blank.

**Comment by 3v1n0** [comment]
Can you please share a sample of your csv file?

Once sanitized of personal info of course.

**Comment by jomajojo** [comment]
my config.ini is :
[n26]
plugin = n26
currency = EUR
bank_id = N26 Bank
account = DE12345678910

Now the filename is "Contoprincipale_2024-09-01_2024-09-23.csv"

[Contoprincipale_2024-09-01_2024-09-23.csv](https://github.com/user-attachments/files/17115347/Contoprincipale_2024-09-01_2024-09-23.csv)

With this file I got the following ofx file
[Contoprincipale_2024-09-01_2024-09-23.ofx.txt](https://github.com/user-attachments/files/17115476/Contoprincipale_2024-09-01_2024-09-23.ofx.txt)

I modified the n26.py the following way. 

[n26.py.txt](https://github.com/user-attachments/files/17115577/n26.py.txt)

It needs fine-tuning because I just ignored some entries.
For this reason, the console shows a lot of warnings like this:
WARNING: Mapping not found for Credit Transfer
WARNING: Mapping not found for Credit Transfer
WARNING: Mapping not found for Presentment
WARNING: Mapping not found for Reward
WARNING: Mapping not found for Credit Transfer
WARNING: Mapping not found for Credit Transfer
WARNING: Mapping not found for Presentment
WARNING: Mapping not found for Debit Transfer

I have more accounts, I need the statements BANKID and ACCTID  are taken from the config.ini 
(generated with " ofxstatement edit-config") . 
This way GnuCash will automatically associate the generated .ofx file with a particular GnuCash account.

**Comment by jomajojo** [comment]
at the moment I have solved the following way 
[n26.py.txt](https://github.com/user-attachments/files/17266964/n26.py.txt)
 for sure as it needs some finetuning.
As I am not a programmer I cannot open a pull request or a fork.
